lib/client-assets.php

```diff
@@ -211,6 +214,27 @@ function wc_admin_print_script_settings() {
 }
 add_action( 'admin_print_footer_scripts', 'wc_admin_print_script_settings', 1 );
 
+/**
+ * Sets default values for user preferences.
+ *
+ * @param array $user_data Array of user data.
+ * @return array Filtered array of user data.
+ */
+function wc_admin_get_user_defaults( $user_data ) {
+	// If no settings for performance indicators are stored, these are the defaults to disable.
+	if ( ! is_array( $user_data['dashboard_performance_indicators'] ) ) {
+		$user_data['dashboard_performance_indicators'] = array(
```

Comment:
Nice! /cc @LevinMedia do these look like good defaults for Performance Indicators?

Reply:
Just a note that these are the ones that are **hidden** by default (per the issue). So

Reply:
This list is now here:
https://github.com/woocommerce/wc-admin/pull/1343/commits/bcca36065894e5f3fda239bb80759e8d23e96468#diff-71321a1ffc66cd298d29c3719d481cd7R171

Reply:
Lets make default excluded summary numbers:

```
'taxes/order_tax',
'taxes/shipping_tax',
'downloads/download_count',
```

that gives us an even 10 in the grid.

Reply:
Done